client/blocks/global-notice/README.md

```diff
@@ -0,0 +1,24 @@
+Global Notice
```

Comment:
We already have `<GlobalNotices />` components. See https://github.com/Automattic/wp-calypso/tree/master/client/components/global-notices. It is a bit different, but I think this change in such shape would introduce some confusion. We should rather put this component in the same folder and pick a different name to make it clear how it differs from we already have.